carp.common/src/commonMain/kotlin/dk/cachet/carp/common/DateTime.kt

```diff
@@ -17,27 +13,22 @@ expect class DateTime( msSinceUTC: Long )
 {
     val msSinceUTC: Long
 
+
     companion object
     {
         fun now(): DateTime
     }
 
+
     /**
      * Output as ISO 8601 UTC date and time in extended format with day and seconds precision with 3 decimal places following a period.
      * E.g., "2020-01-01T12:00:00.000Z"
      */
-    override fun toString(): String
+    fun defaultFormat(): String
```

Comment:
Output to specific and lossy representations should be named after the representation i.e. `toSeconds()`, `toISOString()` etc.

I guess all this will be fixed with kotlinx datetime ;)

Reply:
Solved in https://github.com/cph-cachet/carp.core-kotlin/pull/195/commits/a390e8ce7763a1d22526067a784506b2948a8b93

`toString` is simply used now and ISO is the default.